core/iwasm/libraries/debug-engine/handler.c

```diff
@@ -205,7 +205,7 @@ handle_general_query(WASMGDBServer *server, char *payload)
         os_mutex_lock(&tmpbuf_lock);
         snprintf(tmpbuf, MAX_PACKET_SIZE,
                  "qXfer:libraries:read+;PacketSize=%" PRIx32 ";",
-                 MAX_PACKET_SIZE);
+                 (uint32)MAX_PACKET_SIZE);
```

Comment:
why not change PRIx32 to %xd

Reply:
do you mean `%x`?
it's a valid alternative.
well, theoretically MAX_PACKET_SIZE can be of other type as it's user-overridable.
but i don't have a strong preference for one way or another.

Reply:
> do you mean `%x`?

Yes.

> it's a valid alternative. well, theoretically MAX_PACKET_SIZE can be of other type as it's user-overridable.

I assume MAX_PACKET_SIZE  doesn't add any suffix(the integer constant without any suffix is int).

> but i don't have a strong preference for one way or another.

 I prefer the code doesn't have the cast as much as possible.

Reply:
> > do you mean `%x`?
> 
> Yes.
> 
> > it's a valid alternative. well, theoretically MAX_PACKET_SIZE can be of other type as it's user-overridable.
> 
> I assume MAX_PACKET_SIZE doesn't add any suffix(the integer constant without any suffix is int).
> 
> > but i don't have a strong preference for one way or another.
> 
> I prefer the code doesn't have the cast as much as possible.

ok. i'm fine with either ways. let me change it.

Reply:
done.